sentry_sdk/integrations/launchdarkly.py

```diff
@@ -0,0 +1,73 @@
+from typing import TYPE_CHECKING
+import sentry_sdk
+
+from sentry_sdk.integrations import DidNotEnable, Integration
+
+try:
+    import ldclient
+    from ldclient.hook import Hook, Metadata
+
+    if TYPE_CHECKING:
+        from ldclient import LDClient
+        from ldclient.hook import EvaluationSeriesContext
+        from ldclient.evaluation import EvaluationDetail
+
+        from sentry_sdk._types import Event, ExcInfo
+        from typing import Any, Optional
+except ImportError:
+    raise DidNotEnable("LaunchDarkly is not installed")
+
+
+class LaunchDarklyIntegration(Integration):
+    identifier = "launchdarkly"
+
+    def __init__(self, client=None):
+        # type: (LDClient | None) -> None
+        """
+        @param client    An initialized LDClient instance. If a client is not provided, this integration will attempt to
+                         use the shared global instance. This will fail if ldclient.set_config() hasn't been called.
```

Comment:
Line length and formatting.  AFAIK we don't follow this comment or indentation style.

> This will fail if ldclient.set_config() hasn't been called.

This can be removed.  We don't know this is true for the version they're running.  It could be a different method, it could be automated, etc.

Reply:
I see this comment style used: https://github.com/getsentry/sentry-python/blob/74d8148586621d360912f555c640f1de9a902419/sentry_sdk/api.py#L375-L382